ENGAGEHF.xcodeproj/project.pbxproj

```diff
@@ -511,6 +577,7 @@
 				2FC3439129EE6349002D773C /* AppIcon.png in Resources */,
 				653A255528338800005D4D48 /* Assets.xcassets in Resources */,
 				2FC3439029EE6346002D773C /* SocialSupportQuestionnaire.json in Resources */,
+				4DB025C62BBE21EF002D2545 /* index.js in Resources */,
```

Comment:
We will remove this in this PR but also for future reference: The Firebase cloud functions don't really have to be added to the Xcode project, I would recommend VS Code to edit the JS/TS code.